app/mailers/favorite_mailer.rb

```diff
@@ -12,4 +12,14 @@ def new_comment(user, post, comment)
 
     mail(to: user.email, subject: "New comment on #{post.title}")
   end
+
+  def new_post(post)
+    headers["Message-ID"] = "<posts/#{post.id}@http://localhost:3000/>"
+    headers["In-Reply-To"] = "<post/#{post.id}@http://localhost:3000/>"
+    headers["References"] = "<post/#{post.id}@http://localhost:3000/>"
```

Comment:
Hi Xander, 

If I can recall, they wanted to have a series of 'nested responses', like you get in an email.